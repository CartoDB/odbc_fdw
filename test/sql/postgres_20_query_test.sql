SELECT * FROM postgres_test_table;
SELECT * FROM query_postgres_test_table;
SELECT * FROM existent_table_in_schema_public;
SELECT * FROM test_table_in_schema;
SELECT * FROM ODBCTablesList('postgres_fdw', 1);
SELECT * FROM ODBCTableSize('postgres_fdw', 'postgres_test_table');
SELECT * FROM ODBCQuerySize('postgres_fdw', 'select * from postgres_test_table');
\x
SELECT * FROM postgres_var_test_table;
\x
SELECT * FROM postgres_short_test_table;