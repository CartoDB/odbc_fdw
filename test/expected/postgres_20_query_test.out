SELECT * FROM postgres_test_table;
 id | varchar_example | text_example | integer_example | numeric_example |    timestamp_example     | boolean_example 
----+-----------------+--------------+-----------------+-----------------+--------------------------+-----------------
  1 | example         | example      |             100 |       10.120000 | Fri Jan 01 00:00:00 2016 | t
(1 row)

SELECT * FROM query_postgres_test_table;
 count 
-------
     1
(1 row)

SELECT * FROM existent_table_in_schema_public;
ERROR:  Executing ODBC query to get table size
ERROR: relation "nonexistent_schema.existent_table_in_schema_public" does not exist;
Error while executing the query
SELECT * FROM test_table_in_schema;
 id |  data   
----+---------
  1 | example
(1 row)

SELECT * FROM ODBCTablesList('postgres_fdw', 1);
 schema |              name               
--------+---------------------------------
 public | existent_table_in_schema_public
(1 row)

SELECT * FROM ODBCTableSize('postgres_fdw', 'postgres_test_table');
 odbctablesize 
---------------
             1
(1 row)

SELECT * FROM ODBCQuerySize('postgres_fdw', 'select * from postgres_test_table');
 odbcquerysize 
---------------
             1
(1 row)

\x
SELECT * FROM postgres_var_test_table;
-[ RECORD 1 ]----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
varch  | 123456789abcdef101112131415161718191a1b1c1d1e1f202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f404142434445464748494a4b4c4d4e4f505152535455565758595a5b5c5d5e5f606162636465666768696a6b6c6d6e6f707172737475767778797a7b7c7d7e7f808182838485868788898a8b8c8d8e8f909192939495969798999a9b9c9d9e9fa0a1a2a3a4a5a6a7a8a9aaabacadaeafb0b1b2b3b4b5b6b7b8b9babbbcbdbebfc0c1c2c3c4c5c6c7c8c9cacbcccdcecfd0d1d2d3d4d5d6d7d8d9dadbdcdddedfe0e1e2e3e4e5e6e7e8e9eaebecedeeeff0f1f2f3f4f5f6f7f8f9fafbfcfdfeff10010110210310410510610710810910a10b10c10d10e10f11011111211311411511611711811911a11b11c11d11e11f12012112212312412512612712812912a12b12c12d12e12f13013113213313413513613713813913a13b13c13d13e13f14014114214314414514614714814914a14b14c14d14e14f15015115215315415515615715815915a15b15c15d15e15f16016116216316416516616716816916a16b16c16d16e16f17017117217317417517617717817917a17b17c17d17e17f18018118218318418518618718818918a18b18c18d18e18f19019119219319419519619719819919a19b19c19d19e19f1a01a11a21a31a41a51a61a71a81a91aa1ab1ac1ad1ae1af1b01b11b21b31b41b51b61b71b81b91ba1bb1bc1bd1be1bf1c01c11c21c31c41c51c61c71c81c91ca1cb1cc1cd1ce1cf1d01d11d21d31d41d51d61d71d81d91da1db1dc1dd1de1df1e01e11e21e31e41e51e61e71e81e91ea1eb1ec1ed1ee1ef1f01f11f21f31f41f51f61f71f81f91fa1fb1fc1fd1fe1ff20020120220320420520620720820920a20b20c20d20e20f21021121221321421521621721821921a21b21c21d21e21f22022122222322422522622722822922a22b22c22d22e22f23023123223323423523623723823923a23b23c23d23e23f24024124224324424524624724824924a24b24c24d24e24f25025125225325425525625725825925a25b25c25d25e25f26026126226326426526626726826926a26b26c26d26e26f27027127227327427527627727827927a27b27c27d27e27f28028128228328428528628728828928a28b28c28d28e28f29029129229329429529629729829929a29b29c29d29e29f2a02a12a22a32a42a52a62a72a82a92aa2ab2ac2ad2ae2af2b02b12b22b32b42b52b62b72b82b92ba2bb2bc2bd2be2bf2c02c12c22c32c42c52c62c72c82c92ca2cb2cc2cd2ce2cf2d02d12d22d32d42d52d62d72d82d92da2db2dc2dd2de2df2e02e12e22e32e42e52e62e72e82e92ea2eb2ec2ed2ee2ef2f02f12f22f32f42f52f62f72f82f92fa2fb2fc2fd2fe2ff30030130230330430530630730830930a30b30c30d30e30f31031131231331431531631731831931a31b31c31d31e31f32032132232332432532632732832932a32b32c32d32e32f33033133233333433533633733833933a33b33c33d33e33f34034134234334434534634734834934a34b34c34d34e34f35035135235335435535635735835935a35b35c35d35e35f36036136236336436536636736836936a36b36c36d36e36f37037137237337437537637737837937a37b37c37d37e37f38038138238338438538638738838938a38b38c38d38e38f39039139239339439539639739839939a39b39c39d39e39f3a03a13a23a33a43a53a63a73a83a93aa3ab3ac3ad3ae3af3b03b13b23b33b43b53b63b73b83b93ba3bb3bc3bd3be3bf3c03c13c23c33c43c53c63c73c83c93ca3cb3cc3cd3ce3cf3d03d13d23d33d43d53d63d73d83d93da3db3dc3dd3de3df3e03e13e23e33e43e53e63e73e83e93ea3eb3ec3ed3ee3ef3f03f13f23f33f43f53f63f73f83f93fa3fb3fc3fd3fe3ff40040140240340440540640740840940a40b40c40d40e40f41041141241341441541641741841941a41b41c41d41e41f42042142242342442542642742842942a42b42c42d42e42f43043143243343443543643743843943a43b43c43d43e43f44044144244344444544644744844944a44b44c44d44e44f45045145245345445545645745845945a45b45c45d45e45f46046146246346446546646746846946a46b46c46d46e46f47047147247347447547647747847947a47b47c47d47e47f48048148248348448548648748848948a48b48c48d48e48f49049149249349449549649749849949a49b49c49d49e49f4a04a14a24a34a44a54a64a74a84a94aa4ab4ac4ad4ae4af4b04b14b24b34b44b54b64b74b84b94ba4bb4bc4bd4be4bf4c04c14c24c34c44c54c64c74c84c94ca4cb4cc4cd4ce4cf4d04d14d24d34d44d54d64d74d84d94da4db4dc4dd4de4df4e04e14e24e34e44e54e64e74e84e94ea4eb4ec4ed4ee4ef4f04f14f24f34f44f54f64f74f84f94fa4fb4fc4fd4fe4ff50050150250350450550650750850950a50b50c50d50e50f51051151251351451551651751851951a51b51c51d51e51f52052152252352452552652752852952a52b52c52d52e52f53053153253353453553653753853953a53b53c53d53e53f54054154254354454554654754854954a54b54c54d54e54f55055155255355455555655755855955a55b55c55d55e55f56056156256356456556656756856956a56b56c56d56e56f57057157257357457557657757857957a57b57c57d57e57f58058158258358458558658758858958a58b58c58d58e58f59059159259359459559659759859959a59b59c59d59e59f5a05a15a25a35a45a55a65a75a85a95aa5ab5ac5ad5ae5af5b05b15b25b35b45b55b65b75b85b95ba5bb5bc5bd5be5bf5c05c15c25c35c45c55c65c75c85c95ca5cb5cc5cd5ce5cf5d05d15d25d35d45d55d65d75d85d95da5db5dc5dd5de5df5e05e15e25e35e45e55e65e75e85e95ea5eb5ec5ed5ee5ef5f05f15f25f35f45f55f65f75f85f95fa5fb5fc5fd5fe5ff60060160260360460560660760860960a60b60c60d60e60f61061161261361461561661761861961a61b61c61d61e61f62062162262362462562662762862962a62b62c62d62e62f63063163263363463563663763863963a63b63c63d63e63f64064164264364464564664764864964a64b64c64d64e64f65065165265365465565665765865965a65b65c65d65e65f66066166266366466566666766866966a66b66c66d66e66f67067167267367467567667767867967a67b67c67d67e67f68068168268368468568668768868968a68b68c68d68e68f69069169269369469569669769869969a69b69c69d69e69f6a06a16a26a36a46a56a66a76a86a96aa6ab6ac6ad6ae6af6b06b16b26b36b46b56b66b76b86b96ba6bb6bc6bd6be6bf6c06c16c26c36c46c56c66c76c86c96ca6cb6cc6cd6ce6cf6d06d16d26d36d46d56d66d76d86d96da6db6dc6dd6de6df6e06e16e26e36e46e56e66e76e86e96ea6eb6ec6ed6ee6ef6f06f16f26f36f46f56f66f76f86f96fa6fb6fc6fd6fe6ff70070170270370470570670770870970a70b70c70d70e70f71071171271371471571671771871971a71b71c71d71e71f72072172272372472572672772872972a72b72c72d72e72f73073173273373473573673773873973a73b73c73d73e73f74074174274374474574674774874974a74b74c74d74e74f75075175275375475575675775875975a75b75c75d75e75f76076176276376476576676776876976a76b76c76d76e76f77077177277377477577677777877977a77b77c77d77e77f78078178278378478578678778878978a78b78c78d78e78f79079179279379479579679779879979a79b79c79d79e79f7a07a17a27a37a47a57a67a77a87a97aa7ab7ac7ad7ae7af7b07b17b27b37b47b57b67b77b87b97ba7bb7bc7bd7be7bf7c07c17c27c37c47c57c67c77c87c97ca7cb7cc7cd7ce7cf7d0
varbin | \x123456789abcdef101112131415161718191a1b1c1d1e1f202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f404142434445464748494a4b4c4d4e4f505152535455565758595a5b5c5d5e5f606162636465666768696a6b6c6d6e6f707172737475767778797a7b7c7d7e7f808182838485868788898a8b8c8d8e8f909192939495969798999a9b9c9d9e9fa0a1a2a3a4a5a6a7a8a9aaabacadaeafb0b1b2b3b4b5b6b7b8b9babbbcbdbebfc0c1c2c3c4c5c6c7c8c9cacbcccdcecfd0d1d2d3d4d5d6d7d8d9dadbdcdddedfe0e1e2e3e4e5e6e7e8e9eaebecedeeeff0f1f2f3f4f5f6f7f8f9fafbfcfdfeff10010110210310410510610710810910a10b10c10d10e10f11011111211311411511611711811911a11b11c11d11e11f12012112212312412512612712812912a12b12c12d12e12f13013113213313413513613713813913a13b13c13d13e13f14014114214314414514614714814914a14b14c14d14e14f15015115215315415515615715815915a15b15c15d15e15f16016116216316416516616716816916a16b16c16d16e16f17017117217317417517617717817917a17b17c17d17e17f18018118218318418518618718818918a18b18c18d18e18f19019119219319419519619719819919a19b19c19d19e19f1a01a11a21a31a41a51a61a71a81a91aa1ab1ac1ad1ae1af1b01b11b21b31b41b51b61b71b81b91ba1bb1bc1bd1be1bf1c01c11c21c31c41c51c61c71c81c91ca1cb1cc1cd1ce1cf1d01d11d21d31d41d51d61d71d81d91da1db1dc1dd1de1df1e01e11e21e31e41e51e61e71e81e91ea1eb1ec1ed1ee1ef1f01f11f21f31f41f51f61f71f81f91fa1fb1fc1fd1fe1ff20020120220320420520620720820920a20b20c20d20e20f21021121221321421521621721821921a21b21c21d21e21f22022122222322422522622722822922a22b22c22d22e22f23023123223323423523623723823923a23b23c23d23e23f24024124224324424524624724824924a24b24c24d24e24f25025125225325425525625725825925a25b25c25d25e25f26026126226326426526626726826926a26b26c26d26e26f27027127227327427527627727827927a27b27c27d27e27f28028128228328428528628728828928a28b28c28d28e28f29029129229329429529629729829929a29b29c29d29e29f2a02a12a22a32a42a52a62a72a82a92aa2ab2ac2ad2ae2af2b02b12b22b32b42b52b62b72b82b92ba2bb2bc2bd2be2bf2c02c12c22c32c42c52c62c72c82c92ca2cb2cc2cd2ce2cf2d02d12d22d32d42d52d62d72d82d92da2db2dc2dd2de2df2e02e12e22e32e42e52e62e72e82e92ea2eb2ec2ed2ee2ef2f02f12f22f32f42f52f62f72f82f92fa2fb2fc2fd2fe2ff30030130230330430530630730830930a30b30c30d30e30f31031131231331431531631731831931a31b31c31d31e31f32032132232332432532632732832932a32b32c32d32e32f33033133233333433533633733833933a33b33c33d33e33f34034134234334434534634734834934a34b34c34d34e34f35035135235335435535635735835935a35b35c35d35e35f36036136236336436536636736836936a36b36c36d36e36f37037137237337437537637737837937a37b37c37d37e37f38038138238338438538638738838938a38b38c38d38e38f39039139239339439539639739839939a39b39c39d39e39f3a03a13a23a33a43a53a63a73a83a93aa3ab3ac3ad3ae3af3b03b13b23b33b43b53b63b73b83b93ba3bb3bc3bd3be3bf3c03c13c23c33c43c53c63c73c83c93ca3cb3cc3cd3ce3cf3d03d13d23d33d43d53d63d73d83d93da3db3dc3dd3de3df3e03e13e23e33e43e53e63e73e83e93ea3eb3ec3ed3ee3ef3f03f13f23f33f43f53f63f73f83f93fa3fb3fc3fd3fe3ff40040140240340440540640740840940a40b40c40d40e40f41041141241341441541641741841941a41b41c41d41e41f42042142242342442542642742842942a42b42c42d42e42f43043143243343443543643743843943a43b43c43d43e43f44044144244344444544644744844944a44b44c44d44e44f45045145245345445545645745845945a45b45c45d45e45f46046146246346446546646746846946a46b46c46d46e46f47047147247347447547647747847947a47b47c47d47e47f48048148248348448548648748848948a48b48c48d48e48f49049149249349449549649749849949a49b49c49d49e49f4a04a14a24a34a44a54a64a74a84a94aa4ab4ac4ad4ae4af4b04b14b24b34b44b54b64b74b84b94ba4bb4bc4bd4be4bf4c04c14c24c34c44c54c64c74c84c94ca4cb4cc4cd4ce4cf4d04d14d24d34d44d54d64d74d84d94da4db4dc4dd4de4df4e04e14e24e34e44e54e64e74e84e94ea4eb4ec4ed4ee4ef4f04f14f24f34f44f54f64f74f84f94fa4fb4fc4fd4fe4ff50050150250350450550650750850950a50b50c50d50e50f51051151251351451551651751851951a51b51c51d51e51f52052152252352452552652752852952a52b52c52d52e52f53053153253353453553653753853953a53b53c53d53e53f54054154254354454554654754854954a54b54c54d54e54f55055155255355455555655755855955a55b55c55d55e55f56056156256356456556656756856956a56b56c56d56e56f57057157257357457557657757857957a57b57c57d57e57f58058158258358458558658758858958a58b58c58d58e58f59059159259359459559659759859959a59b59c59d59e59f5a05a15a25a35a45a55a65a75a85a95aa5ab5ac5ad5ae5af5b05b15b25b35b45b55b65b75b85b95ba5bb5bc5bd5be5bf5c05c15c25c35c45c55c65c75c85c95ca5cb5cc5cd5ce5cf5d05d15d25d35d45d55d65d75d85d95da5db5dc5dd5de5df5e05e15e25e35e45e55e65e75e85e95ea5eb5ec5ed5ee5ef5f05f15f25f35f45f55f65f75f85f95fa5fb5fc5fd5fe5ff60060160260360460560660760860960a60b60c60d60e60f61061161261361461561661761861961a61b61c61d61e61f62062162262362462562662762862962a62b62c62d62e62f63063163263363463563663763863963a63b63c63d63e63f64064164264364464564664764864964a64b64c64d64e64f65065165265365465565665765865965a65b65c65d65e65f66066166266366466566666766866966a66b66c66d66e66f67067167267367467567667767867967a67b67c67d67e67f68068168268368468568668768868968a68b68c68d68e68f69069169269369469569669769869969a69b69c69d69e69f6a06a16a26a36a46a56a66a76a86a96aa6ab6ac6ad6ae6af6b06b16b26b36b46b56b66b76b86b96ba6bb6bc6bd6be6bf6c06c16c26c36c46c56c66c76c86c96ca6cb6cc6cd6ce6cf6d06d16d26d36d46d56d66d76d86d96da6db6dc6dd6de6df6e06e16e26e36e46e56e66e76e86e96ea6eb6ec6ed6ee6ef6f06f16f26f36f46f56f66f76f86f96fa6fb6fc6fd6fe6ff70070170270370470570670770870970a70b70c70d70e70f71071171271371471571671771871971a71b71c71d71e71f72072172272372472572672772872972a72b72c72d72e72f73073173273373473573673773873973a73b73c73d73e73f74074174274374474574674774874974a74b74c74d74e74f75075175275375475575675775875975a75b75c75d75e75f76076176276376476576676776876976a76b76c76d76e76f77077177277377477577677777877977a77b77c77d77e77f78078178278378478578678778878978a78b78c78d78e78f79079179279379479579679779879979a79b79c79d79e79f7a07a17a27a37a47a57a67a77a87a97aa7ab7ac7ad7ae7af7b07b17b27b37b47b57b67b77b87b97ba7bb7bc7bd7be7bf7c07c17c27c37c47c57c67c77c87c97ca7cb7cc7cd7ce7cf7d0

\x
